### PR TITLE
 test cexio for cancelOrder params (CancelOrderCurrencyPair)

### PR DESCRIPTION
**Pull request by Jevgeijs**

### CODE REVIEW

File: xchange-cexio/src/test/java/org/knowm/xchange/cexio/service/TradeServiceIntegration.java [hunk 1]
```diff
@@ -86,6 +87,32 @@ public void orderPlaceGetCancelTest() throws IOException, InterruptedException {
         "Returned order must be canceled", orders.get(0).getStatus() == Order.OrderStatus.CANCELED);
   }
 
+  @Test
+  public void cancelAllOrdersTest() throws IOException, InterruptedException {
```
Comment (sergluka) [inline xchange-cexio/src/test/java/org/knowm/xchange/cexio/service/TradeServiceIntegration.java]: A name is misleading. This test check cancel by a pair.

File: xchange-cexio/src/test/java/org/knowm/xchange/cexio/service/TradeServiceIntegration.java [hunk 2]
```diff
@@ -86,6 +87,32 @@ public void orderPlaceGetCancelTest() throws IOException, InterruptedException {
...
+    String orderId2 = tradeService.placeLimitOrder(order);
+
+    tradeService.cancelOrder(new CancelOrderByCurrencyPair() {
+      @Override
+      public CurrencyPair getCurrencyPair() {
+        return new CurrencyPair("BCH/USD");
+      }
+    });
+
+    List<Order> orders = (List<Order>) tradeService.getOrder(orderId, orderId2);
```
Comment (sergluka) [inline xchange-cexio/src/test/java/org/knowm/xchange/cexio/service/TradeServiceIntegration.java:97]: This cast is unnecessary.

File: xchange-cexio/src/test/java/org/knowm/xchange/cexio/service/TradeServiceIntegration.java [hunk 3]
```diff
@@ -40,9 +40,9 @@ public void setup() throws IOException {
 
     order =
         buildOrder(
-            Order.OrderType.BID,
+            Order.OrderType.ASK,
```
Comment (sergluka) [inline xchange-cexio/src/test/java/org/knowm/xchange/cexio/service/TradeServiceIntegration.java]: What the reason of this test data change?

File: xchange-cexio/src/main/java/org/knowm/xchange/cexio/service/CexIOTradeService.java [hunk 4]
```diff
@@ -58,21 +56,26 @@ public OpenOrders getOpenOrders(OpenOrdersParams params) throws IOException {
 
   @Override
   public String placeMarketOrder(MarketOrder marketOrder) throws IOException {
+      /*
+    Only in market order!
+    Presently, the exchanger is designed in such way that, depending on the BID/ASK the currency changes
```
Comment (sergluka) [inline xchange-cexio/src/main/java/org/knowm/xchange/cexio/service/CexIOTradeService.java]: exchanger -> exchange

File: xchange-cexio/src/main/java/org/knowm/xchange/cexio/service/CexIOTradeService.java [hunk 5]
```diff
@@ -58,21 +56,26 @@ public OpenOrders getOpenOrders(OpenOrdersParams params) throws IOException {
...
+      (accordingly, you must specify the amount in another currency)
+    Example: CurrencyPair.BCH_USD, Order.OrderType.ASK, Amount = 0.02 (BCH)
+    Example: CurrencyPair.BCH_USD, Order.OrderType.BID, Amount = 20 (USD)
+    Сurrently cannot be implemented!
+    */
 
     throw new NotAvailableFromExchangeException();
```
Comment (sergluka) [inline xchange-cexio/src/main/java/org/knowm/xchange/cexio/service/CexIOTradeService.java]: `NotYetImplementedForExchangeException` suites better.

**Review by sergluka — DISMISSED** [review]

File: xchange-cexio/src/test/java/org/knowm/xchange/cexio/service/TradeServiceIntegration.java (hunk 3 above)
Comment (Jevgeijs) [reply xchange-cexio/src/test/java/org/knowm/xchange/cexio/service/TradeServiceIntegration.java]: as there was no money in the wallet to BID, but I shouldn't commit it. I understand.

**Pull request merged**